server/data/welcomeClient.test.ts

```diff
@@ -135,6 +136,23 @@ describe('welcomeClient', () => {
     })
   })
 
+  describe('confirmTemporaryAbsence', () => {
+    const prisonNumber = 'A1234AB'
+    const agencyId: AgencyId = {
```

Comment:
The request object is more complex where AgancyId is one of the parameters maybe will be better to rename it to ConfirmTemporaryAbsenceRequest.